pipeline/outputs/loki.md

```diff
@@ -10,28 +10,29 @@ Be aware there is a separate Golang output plugin provided by [Grafana](https://
+| http\_passwd                    | Set HTTP basic authentication password                                                                                                                                                                                                                                                                                                                                                                                          |                   |
+| bearer\_token                   | Set bearer token authentication token value.                                                                                                                                                                                                                                                                                                                                                                                    |                   |
+| header                          | Add additional arbitrary HTTP header key/value pair. Multiple headers can be set.                                                                                                                                                                                                                                                                                                                                               |                   |
+| tenant\_id                      | Tenant ID used by default to push logs to Loki. If omitted or empty it assumes Loki is running in single-tenant mode and no X-Scope-OrgID header is sent.                                                                                                                                                                                                                                                                       |                   |
+| labels                          | Stream labels for API request. It can be multiple comma separated of strings specifying  `key=value` pairs. In addition to fixed parameters, it also allows to add custom record keys \(similar to `label_keys` property\). More details in the Labels section.                                                                                                                                                                 | job=fluent-bit    |
+| label\_keys                     | Optional list of record keys that will be placed as stream labels. This configuration property is for records key only. More details in the Labels section.                                                                                                                                                                                                                                                                     |                   |
```

Comment:
```suggestion
| `uri`                             | Specify a custom HTTP URL. It must start with forward slash. | `/loki/api/v1/push` |
| `port`                            | Loki TCP port. | `3100` |
| `tls`                             | Use TLS authentication. | `off` |
| `http_user`                       | Set HTTP basic authentication user name. | _none_ |
| `http_passwd`                     | Set HTTP basic authentication password. | _none_ |
| `bearer_token`                    | Set bearer token authentication token value. | _none_ |
| `header`                          | Add additional arbitrary HTTP header key/value pair. Multiple headers can be set. | _none_ |
| `tenant_id`                       | Tenant ID used by default to push logs to Loki. If omitted or empty it assumes Loki is running in single-tenant mode and no X-Scope-OrgID header is sent. | _none_ |
| `labels`                          | Stream labels for API request. It can be multiple comma separated of strings specifying `key=value` pairs. In addition to fixed parameters, it also allows to add custom record keys (similar to `label_keys` property). More details in the Labels section. | `job=fluent-bit` |
| `label_keys`                      | Optional list of record keys that will be placed as stream labels. This configuration property is for records key only. More details in the Labels section. | _none_ |
```